lock-api/src/main/java/com/palantir/lock/v2/TimelockService.java

```diff
@@ -71,10 +71,31 @@ default boolean isInitialized() {
     @Path("refresh-locks")
     Set<LockToken> refreshLockLeases(Set<LockToken> tokens);
 
+    /**
+     * Releases locks associated with the set of {@link LockToken}s provided.
+     * The set of tokens returned are the tokens for which the associated locks were unlocked in this call.
+     * It is possible that a token that was provided is NOT in the returned set (e.g. if it expired).
+     * However, in this case it is guaranteed that that token is no longer valid.
```

Comment:
We don't actually use the output in Atlas. It can be useful to distinguish this if you wanted to do a 'check I still have the locks, and then release them' type of operation - this was considered as a possible tx perf improvement (and was going to be used as the next step on the #3282 route)